node/src/Worker.ts

```diff
@@ -629,17 +673,42 @@ export class Worker extends EnhancedEventEmitter<WorkerEvents>
 			throw new TypeError('if given, appData must be an object');
 		}
 
-		const reqData =
+		// Build the request.
+		const fbsListenInfos:FbsWebRtcServer.ListenInfoT[] = [];
+
+		for (const listenInfo of listenInfos)
 		{
-			webRtcServerId : uuidv4(),
-			listenInfos
-		};
+			// NOTE: This check should not be needed since
+			// FbsWebRtcServer.ListenInfoT should throw.
+			// In absence of 'ip' it does not throw in Linux.
+			// TODO: Remove once https://github.com/google/flatbuffers/issues/7739
```

Comment:
Such a PR is merged. Waiting for release?

Reply:
I did want to do it, sorry if I failed.

Reply:
I've just upgraded flatbuffers. I'll remove these next.

Reply:
done.